src/main/java/bio/terra/service/filedata/google/firestore/FireStoreDao.java

```diff
@@ -561,8 +561,10 @@ FireStoreDirectoryEntry computeDirectory(
 
     for (FireStoreDirectoryEntry dirItem : enumComputed) {
       totalSize = totalSize + dirItem.getSize();
-      crc32cCollection.add(StringUtils.lowerCase(dirItem.getChecksumCrc32c()));
-      if (dirItem.getChecksumMd5() != null) {
+      if (!StringUtils.isEmpty(dirItem.getChecksumCrc32c())) {
+        crc32cCollection.add(StringUtils.lowerCase(dirItem.getChecksumCrc32c()));
```

Comment:
There's no need to use `StringUtils` here, as we know the string isn't `null` at this point
```suggestion
        crc32cCollection.add(dirItem.getChecksumCrc32c().toLowerCase());
```

Reply:
It actually might be.  Azure returns this as a null and I want to defend against changes in how we deal with this field.  This seemed much safer

Reply:
Oh wait, nevermind, I was looking at the wrong line.  I see what you mean  great point.  I'll update